code/__DEFINES/atmospherics/atmos_core.dm

```diff
@@ -36,6 +36,8 @@
 #define T20C 293.15
 /// -14C - Temperature used for kitchen cold room, medical freezer, etc.
 #define COLD_ROOM_TEMP 259.15
+/// 1500degK
+#define T1500K 1500
```

Comment:
This doesn't look important enough to be its own define in atmos_core itself.

Reply:
This was done at request of Willard, I will remove if it he says it should be removed or if you think I should just remove it.